### PR TITLE
[ASM] EXPANDR-10510 Updated ServiceNow playbook

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
[EXPANDR-10510](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10510)

## Description
Updating ServiceNow playbook to add additional details to description.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg, please review

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
@capanw , not sure about that validation error.  I added some testing comments in the Jira, please take care and let me know if you have questions.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_57.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Playbooks
+
+##### Cortex ASM - ServieNow Notification
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_57.md:3]:
```suggestion
##### Cortex ASM - ServiceNow Notification
```
Small typo

**Review by johnnywilkes — CHANGES_REQUESTED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml [hunk 2]
```diff
@@ -198,12 +221,69 @@ tasks:
...
+
+          Alert ID: ${alert.id}
+
+          Severity: ${alert.severity}
+
+          IP address: ${alert.remoteip}
+
+          Hostname: ${alert.hostname}
+
+          Tags: ${alert.tags}
```
[the remark this answers is not in the document]
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml:258]: Hey @BigEasyJ, Thanks for reviewing. Utkarsh requested to just have EV2 tags. So, just added them.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml [hunk 3]
```diff
@@ -229,3 +316,14 @@ outputs: []
 tests:
 - No tests (auto formatted)
 fromversion: 6.9.0
+inputSections:
+- description: Generic group for inputs
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml:320]:
```suggestion
- description: Generic group for inputs.
```

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml [hunk 4]
```diff
@@ -229,3 +316,14 @@ outputs: []
...
+  inputs:
+  - OwnerNotificationBody
+  - NotificationTicketType
+  - RemediationGuidance
+  name: General (Inputs group)
+outputSections:
+- description: Generic group for outputs
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ServiceNow_Notification.yml:327]:
```suggestion
- description: Generic group for outputs.
```

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_57.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#### Playbooks
+
+##### Cortex ASM - ServiceNow Notification
+
+Updated the playbook to add additional information to ServiceNow ticket in description.
```
Comment (ShirleyDenkberg) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_57.md:5]:
```suggestion
Updated the playbook to add additional information to the description in the ServiceNow ticket.
```

**Review by johnnywilkes — APPROVED** [review]
Thanks for the updates

**Comment by ShirleyDenkberg** [comment]
@BigEasyJ @johnnywilkes @tcarmeli1 @tomer-pan Doc review completed.

**Comment by capanw** [comment]
@tcarmeli1 @tomer-pan Approved from our side. Please review and merge, please. Thanks

**Comment by capanw** [comment]
@tcarmeli1 @tomer-pan Please let me know if there is anything that is pending for merging. Thank you!!!

**Comment by tcarmeli1** [comment]
Hi @capanw ,
Great work!
Thank you for your contribution.
I am now working on merging this PR.
Thank you again.

**Review by tcarmeli1 — APPROVED** [review]
Great work!

**Pull request merged**